### PR TITLE
Task/numbers - apidocs - inserviceNumbers

### PR DESCRIPTION
**Pull request by xxlaykxx**
GET /accounts/{accountId}/inserviceNumbers/{tn}
GET /accounts/{accountId}/inserviceNumbers
GET /accounts/{accountId}/inserviceNumbers/totals
Снимок экрана 2022-04-10 в 14 16 39

### CODE REVIEW

**Review by mrmeyers99 — APPROVED** [review]
It would be nice to make the changes for referencing other parameter names, but other than that it looks good.

File: site/specs/numbers.json [hunk 1]
```diff
@@ -6320,140 +6320,133 @@
...
-              "type": "string",
-              "default": null
+              "type": "string"
             },
             "example": "TX"
           },
           {
             "name": "ratecenter",
-            "description": "<p>A Rate Center search filter - all returned TNs will be in the entered Rate Center.  This Rate Center must be combined with a State in the query.  If State is not included in the query then the queryu will fail.</p>\n",
+            "description": "A Rate Center search filter - all returned TNs will be in the entered Rate Center.  This Rate Center must be combined with a State in the query.  If State is not included in the query then the query will fail.",
```
[the remark this answers is not in the document]
Comment (xxlaykxx) [reply site/specs/numbers.json:6401]: @mrmeyers99 fixed

File: site/specs/numbers.json [hunk 2]
```diff
@@ -6320,140 +6320,133 @@
         "tags": [
           "/accounts"
         ],
-        "description": "<p>Retrieves a list of in-service phone numbers associated with the account ID. There are multiple search parameters for searching for in-service numbers:</p>\n\n<ul>\n    <li>size and page for pagination</li>\n    <li>area code</li>\n    <li>Npa-Nxx</li>\n    <li>LATA</li>\n    <li>state</li>\n    <li>rate center</li>\n</ul>\n",
-        "operationId": "GET /accounts/{accountId}/inserviceNumbers",
+        "description": "Retrieves a list of in-service phone numbers associated with the account ID",
+        "summary": "Fetch in-service phone numbers.",
```
Comment (mrmeyers99) [inline site/specs/numbers.json:6324]:
```suggestion
        "summary": "Fetch in-service phone numbers",
```

The summary is what the operation will be named on the UI, I don't think it should have a period in it.

Comment (xxlaykxx) [reply site/specs/numbers.json:6324]: fixed

**Pull request merged**